Binary files gtkglext-1.0.0.orig/examples/coolwave and gtkglext-1.0.0/examples/coolwave differ
diff -Nurd -x'*~' gtkglext-1.0.0.orig/gdk/x11/gdkglquery-x11.c gtkglext-1.0.0/gdk/x11/gdkglquery-x11.c
--- gtkglext-1.0.0.orig/gdk/x11/gdkglquery-x11.c	2003-05-15 16:21:58.000000000 -0400
+++ gtkglext-1.0.0/gdk/x11/gdkglquery-x11.c	2003-05-15 21:31:57.000000000 -0400
@@ -219,6 +219,7 @@
   GdkGLProc proc_address = NULL;
 
   GDK_GL_NOTE (FUNC, g_message (" - gdk_gl_get_proc_address ()"));
+  file_name = NULL;
 
   if (!initialized)
     {
@@ -226,7 +227,6 @@
        * Look up glXGetProcAddress () function.
        */
 
-      file_name = g_module_build_path (NULL, "GL");
 
       GDK_GL_NOTE (MISC, g_message (" - Open %s", file_name));
 
@@ -252,7 +252,6 @@
           g_warning ("Cannot open %s", file_name);
         }
 
-      g_free (file_name);
 
       initialized = TRUE;
     }
@@ -275,7 +274,6 @@
       /* Try g_module_symbol () */
 
       /* libGL */
-      file_name = g_module_build_path (NULL, "GL");
 
       GDK_GL_NOTE (MISC, g_message (" - Open %s", file_name));
 
@@ -294,12 +292,10 @@
           g_warning ("Cannot open %s", file_name);
         }
 
-      g_free (file_name);
 
       if (proc_address == NULL)
         {
           /* libGLcore */
-          file_name = g_module_build_path (NULL, "GLcore");
 
           GDK_GL_NOTE (MISC, g_message (" - Open %s", file_name));
 
@@ -314,13 +310,11 @@
               g_module_close (module);
             }
 
-          g_free (file_name);
         }
     }
   else
     {
       /* libGLU */
-      file_name = g_module_build_path (NULL, "GLU");
 
       GDK_GL_NOTE (MISC, g_message (" - Open %s", file_name));
 
@@ -339,7 +333,6 @@
           g_warning ("Cannot open %s", file_name);
         }
 
-      g_free (file_name);
     }
 
   return proc_address;
